modules/deploy/pages/deployment-option/self-hosted/manual/production/production-readiness.adoc

```diff
@@ -0,0 +1,668 @@
+
+[.side-by-side]
+--
+.Input
+[source,bash]
+---- 
+rpk cluster info
+----
+
+.Output
+[source,bash]
+---- 
+CLUSTER
+=======
+redpanda.be267958-279d-49cd-ae86-98fc7ed2de48
+
+BROKERS
+=======
+ID    HOST            PORT  RACK
+0*    54.70.51.189    9092  us-west-2a
+1     35.93.178.18    9092  us-west-2b
+2     35.91.121.126   9092  us-west-2c
+----
+--
+
+Confirm that xref:manage:rack-awareness.adoc#configure-rack-awareness[rack awareness] is configured correctly.
+
+[.side-by-side]
+--
+.Input
+[source,bash]
+----
+rpk cluster config get enable_rack_awareness
+----
+
+.Output
+[source,bash]
+---- 
+true
+----
+--
+
+See also:
+
+* xref:deploy:deployment-option/self-hosted/manual/high-availability.adoc#multi-az-deployments[Multi-AZ deployments]
+* xref:manage:kubernetes/k-rack-awareness.adoc#configure-rack-awareness[Configure rack awareness in Kubernetes]
+
+== Level 3 production readiness
+
+The Level 3 readiness checklist helps you to confirm that:
+
+- You are proactively monitoring mission-critical workloads, business continuity solutions, and integration into enterprise security systems
+- Your enterprise is ready to run mission-critical workloads
+
+Level 3 production readiness ensures full enterprise readiness, which indicates that your system is operating at the highest levels of availability and can prevent and recover from the most serious of incidents.
+
+=== Configure alerts 
+A standard set of alerts for xref:manage:monitoring.adoc#generate-grafana-dashboard[Grafana] or xref:manage:monitoring.adoc#configure-prometheus[Prometheus] is provided. However, you should customize these alerts for your specific needs. 
+
+See also: xref:reference:monitor-metrics.adoc[Monitoring Metrics]
+
+=== Backup and disaster recovery (DR) solution
+Confirm that you have a backup and disaster recovery (DR) solution in place. You can configure backup and restore using xref:manage:whole-cluster-restore.adoc[Tiered Storage Whole Cluster Recovery].
+
+Be sure to confirm that the backup and DR solution has been tested.
+
+In the case of disaster recovery, confirm that a standby cluster is configured and running with replication (such as xref:upgrade:migrate/data-migration.adoc[MirrorMaker2]). Also verify that monitoring is configured to ensure that MirrorMaker2 is running and keeping up with replication traffic.
+
+=== Deployment automation
+Review your deployment automation. Specifically, if you need to reprovision a cluster, ensure that cluster installation is managed using automation such as xref:deploy:deployment-option/self-hosted/manual/production/production-deployment-automation.adoc#use-terraform-to-set-up-infrastructure[Terraform], xref:deploy:deployment-option/self-hosted/manual/production/production-deployment-automation.adoc#use-ansible-to-install-redpanda[Ansible], or xref:manage:kubernetes/k-configure-helm-chart.adoc[Helm], and that the configuration is saved in source control.
```

Comment:
```suggestion
Review your deployment automation. Specifically, if you need to reprovision a cluster, ensure that cluster installation is managed using automation such as xref:deploy:deployment-option/self-hosted/manual/production/production-deployment-automation.adoc#use-terraform-to-set-up-infrastructure[Terraform], xref:deploy:deployment-option/self-hosted/manual/production/production-deployment-automation.adoc#use-ansible-to-install-redpanda[Ansible], or xref:manage:kubernetes/k-configure-helm-chart.adoc[Helm], and that the configuration is saved in source control.
```

Reply:
@micheleRP No changes here. ??

Reply:
@Feediver1 sorry, this is good. I'll delete the non-suggestion!